pages/your_domain/bioimaging_data.md

```diff
@@ -23,19 +23,22 @@ Bioimaging specialists are acquiring an ever growing amount of data: images, ass
-Understanding and capturing the metadata are critical for many reasons, just to mention a few: analysis, detection of possible faults in acquisition systems. It is important to decide how much details will be recorded since this could dramatically increase the metadata volume and therefore the effort required to capture the metadata.
-
-The collection of images can take several forms: 
- - Data acquired within a facility
- - Data acquired in other facility (commissioned work or external guest user) and "transported" by the users to their facility
- - Slides scanned for example.
 
+The number image proprietary formats is very large. It is a difficult challenge to support i.e. read/extract metadata.
+The [Bio-formats](https://bio-formats.readthedocs.io/) library currently supports over [150 different file formats](https://bio-formats.readthedocs.io/en/latest/supported-formats.html).
+The [Dataset Structure Table](https://bio-formats.readthedocs.io/en/latest/formats/dataset-table.html) shows the extension of the files to read and indicates the structure of the image itself e.g. single file, multiple files, one image file and a companion file, etc.
 
 ### Data management challenges
 
-The number of files and the size of files could be extremely large. Deleting/misplacing a file could invalidate the study itself, preventing its reuse. Managing images immediately becomes a major problem, not only the binary files need to be handled, but also the associated metadata. Several efforts have been made and are still ongoing to capture those metadata. Understanding and capturing the metadata are critical for many reasons, just to mention a few: analysis, detection of possible faults in acquisition systems. It is important to decide how much details will be recorded since this could dramatically increase the metadata volume and therefore the effort required to capture the metadata.
+The number of files and the size of files could be extremely large. Deleting/misplacing a file could invalidate the study itself, preventing its reuse. 
 
+Managing images immediately becomes a larger problem, not only the binary files need to be handled, but also the associated metadata. Several efforts have been made and still ongoing to capture those metadata.
+Understanding and capturing the metadata are critical for many reasons, just to mention a few: analysis, detection of possible faults in acquisition systems. It is important to decide how much details will be recorded since this could dramatically increase the metadata volume and therefore the effort required to capture the metadata.
+
+The collection of images can take several forms: 
+ - data acquired within a facility;
+ - data acquired in other facility (commissioned work or external guest user) and "transported" by the users to their facility;
```

Comment:
It is not immediately clear why the "form" would be different depending on the facility. 
More details should be provided to further enrich this list.